components/event-publisher-proxy/pkg/cloudevents/utils.go

```diff
@@ -10,6 +10,10 @@ import (
 	cehttp "github.com/cloudevents/sdk-go/v2/protocol/http"
 )
 
+const (
+	CEStructuredMode = "application/cloudevents+json"
```

Comment:
Instead of redefining, can we `sdkgocloudevents.ApplicationCloudEventsJSON` where the import is: `sdkgocloudevents "github.com/cloudevents/sdk-go/pkg/cloudevents"`

Reply:
Done